src/test/java/com/example/onlinebookstore/repository/OrderItemRepositoryTest.java

```diff
@@ -0,0 +1,91 @@
+package com.example.onlinebookstore.repository;
+
+import com.example.onlinebookstore.model.Book;
+import com.example.onlinebookstore.model.Order;
+import com.example.onlinebookstore.model.OrderItem;
+import com.example.onlinebookstore.model.User;
+import com.example.onlinebookstore.repository.orderitem.OrderItemRepository;
+import java.math.BigDecimal;
+import java.util.List;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.test.context.jdbc.Sql;
+
+@DataJpaTest
+@AutoConfigureTestDatabase(replace = AutoConfigureTestDatabase.Replace.NONE)
+class OrderItemRepositoryTest {
+    private static final Long ID_TWO = 2L;
+    private static final Long ID_FOUR = 4L;
+    private static final Long ID_FIVE = 5L;
+    @Autowired
+    private OrderItemRepository itemRepository;
+
+    @Test
+    @Sql(scripts = "classpath:database/user/add-users.sql",
+            executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD)
+    @Sql(scripts = "classpath:database/book/add-five-books.sql",
+            executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD)
+    @Sql(scripts = "classpath:database/order/add-orders.sql",
+            executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD)
+    @Sql(scripts = "classpath:database/orderitem/add-order-items.sql",
+            executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD)
+    @Sql(scripts = "classpath:database/orderitem/delete-order-items.sql",
+            executionPhase = Sql.ExecutionPhase.AFTER_TEST_METHOD)
+    @Sql(scripts = "classpath:database/order/delete-orders.sql",
+            executionPhase = Sql.ExecutionPhase.AFTER_TEST_METHOD)
+    @Sql(scripts = "classpath:database/user/delete-users.sql",
+            executionPhase = Sql.ExecutionPhase.AFTER_TEST_METHOD)
+    @Sql(scripts = "classpath:database/book/delete-books.sql",
+            executionPhase = Sql.ExecutionPhase.AFTER_TEST_METHOD)
```

Comment:
Please combine scripts to have 2 annotations instead of 8, like
@Sql(scripts = {"/scripts/entity_test_data.sql", "/scripts/entity_test_data_many.sql"}, 
     executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD)

recheck all classes for same issue